### PR TITLE
Use separate Travis build jobs to run e2e tests and PHPCS

### PR DESCRIPTION
**Pull request by rodrigoprimo**
### Changes proposed in this Pull Request:

This PR creates two new Travis build jobs, one to run the e2e tests and another to run PHPCS. Doing this, instead of running those two checks in the same build job as the PHP 7.2 unit tests, should make it easier to see why a given build failed and it should make the total build time shorter as PHPCS, e2e tests and unit tests will run in parallel.

### How to test the changes in this Pull Request:

1. Make sure that all build jobs pass and that they execute what they are supposed to execute.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +28,9 @@ matrix:
   - php: 5.2
     dist: precise
   - php: 7.2
-    env: WP_VERSION=latest WP_MULTISITE=0 RUN_PHPCS=1 RUN_E2E=1
+    env: WP_VERSION=latest WP_MULTISITE=0 RUN_PHPCS=1
```
Comment (peterfabian) [inline .travis.yml]: This gets confusing in the Travis output when glancing, I would use `name` config property to include a description of the test. E.g. like in wc-admin: https://github.com/woocommerce/wc-admin/blob/7cd132211609cc4ecf3cf47d211c6905e1d3b52a/.travis.yml#L7

**Review by peterfabian — CHANGES_REQUESTED** [review]
Just one small note, otherwise it's fine.

File: .travis.yml (hunk 1 above)
Comment (rodrigoprimo) [reply .travis.yml]: That is great. I didn't know about the `name` config property. I will create a new commit to add it. Thanks.

Comment (rodrigoprimo) [reply .travis.yml]: Commit 87e47cd adds the `name` config property as suggested to identify the build jobs used to run unit tests code coverage, PHPCS and e2e tests.

**Review by peterfabian — APPROVED** [review]
Looking good, thanks!

**Pull request merged**